distribution/linux/openrct2.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Exec=openrct2
+Icon=openrct2
+Name=OpenRCT2
+Comment=Open Roller Coaster Tycoon 2
```

Comment:
@IntelOrca : I fail to see how you would conclude that from what I said. Our own readme calls it a re-implementation, and we're under no obligation to use the same description as we used in the headers.